[2024-05-21T14:41:22.041+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:41:22.064+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-05-02T00:00:00+00:00 [queued]>
[2024-05-21T14:41:22.073+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-05-02T00:00:00+00:00 [queued]>
[2024-05-21T14:41:22.074+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:41:22.092+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2023-05-02 00:00:00+00:00
[2024-05-21T14:41:22.099+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9121) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:41:22.100+0000] {standard_task_runner.py:63} INFO - Started process 9128 to run task
[2024-05-21T14:41:22.100+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'generate_random_numbers', 'scheduled__2023-05-02T00:00:00+00:00', '--job-id', '3213', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpyoxuk2m2']
[2024-05-21T14:41:22.102+0000] {standard_task_runner.py:91} INFO - Job 3213: Subtask generate_random_numbers
[2024-05-21T14:41:22.157+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2023-05-02T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:41:22.275+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2023-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-02T00:00:00+00:00'
[2024-05-21T14:41:22.276+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:41:22.305+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:41:22.307+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:41:22.317+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=generate_random_numbers, run_id=scheduled__2023-05-02T00:00:00+00:00, execution_date=20230502T000000, start_date=20240521T144122, end_date=20240521T144122
[2024-05-21T14:41:22.355+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:41:22.364+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
